bots/happinessncheer.bot.js

```diff
@@ -39,16 +39,24 @@ const getSuggester = (offset=0) =>{
 	const delta = differenceInCalendarDays(now, yearStart);
 	const targetIdx = (delta + offset) % peeps.length
 	return peeps[(targetIdx + PeepOffset) % peeps.length];
-}
+};
+
+const mention = (user)=>{
```

Comment:
You can actually call `mention` on a `map` on the initial array and just use that, instead of calling `mention` each time.

Reply:
oh good point!